connectors/.eslintrc.cjs

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require("path");
+
+/** @type {import("eslint").Linter.Config} */
+const config = {
```

Comment:
Can you align with the config in `/front` ? Eg extending `"prettier"` + the import sorting stuff